packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -15,6 +15,7 @@
   box-sizing: border-box;
   display: flex;
   position: relative;
+  border: 1px solid transparent;
```

Comment:
@micahgodbolt So actually outline is not working as we would hope. Since the width of the box is 100%, the left and right outlines do not display. Only the top and bottom outlines show.

Good news -- we are using box-size: border-box. Which means: The width and height properties (and min/max properties) includes content, padding and border, but not the margin. So if the developer were to spec a height, borders would be takin into account as desired.

Leaving as is.